docs/configure-and-manage/manage/qlight-node.md

```diff
@@ -14,41 +14,42 @@ The server node is usually set up to support [multiple private states](../../con
 
```

Comment:
- Maybe change the heading to "Configure a GoQuorum qlight node" and move this page to the "Configure" section. (In general, use simple present tense for verb headings.)
- Link to this how-to page from the qlight concept page.
- Edit the note in lines 10-11: "Setting up a server node to support MPS is not required, for example, when setting up a qlight client for offloading processing from a full node."

Reply:
Updated